libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/common/SW360Utils.java

```diff
@@ -360,20 +362,25 @@ public static String fieldValueAsString(Object fieldValue) {
             return nullToEmpty((String) fieldValue);
         }
         if (fieldValue instanceof Map) {
-            List<String> mapEntriesAsStrings = nullToEmptyMap(((Map<String, Object>) fieldValue)).entrySet().stream()
-                    .map(e -> {
-                        String valueString = e.getValue() != null ? e.getValue().toString():"";
-                        return e.getKey() + " : " + valueString;
-                    })
-                    .collect(Collectors.toList());
-            return joinStrings(mapEntriesAsStrings);
+            Map<String, Object> originalMap = nullToEmptyMap(((Map<String, Object>) fieldValue));
+            Map<String, String> map = Maps.transformValues(originalMap, v -> v != null ? v.toString() : "");
```

Comment:
the function you have implemented here as lambda is already in the system as `nullToEmptyString`